doc/UsersGuide/source/api/listModel.rst

```diff
@@ -0,0 +1,31 @@
+#CAPTION#
+saveModel
+---------
+
+Lists the contents of a composite model.
+Memory is allocated for contents. The caller is responsible to call free on it.
```

Comment:
I think we should add a comment that the Lua binding takes care of the memory and the caller doesn't need to call free on it.

Reply:
How do we handle this for python? I guess ctypes automatically free the memory, right?

Reply:
In Python this is not automatically done, we have to free the memory manually, the suggestion i read is to add a C function which deallocates the memory something like below

https://stackoverflow.com/questions/4551436/does-ctypes-structures-and-pointers-automatically-free-the-memory-when-the-pytho

Reply:
So we need to add an API function to free the memory?

Reply:
Yes